src/TodoItem.tsx

```diff
@@ -0,0 +1,45 @@
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+// import { TodoLoader } from './TodoLoader';
+
+type Props = {
+  todo: Todo;
+  todos: Todo[];
+  setTodos: (todo: Todo[]) => void;
```

Comment:
let's pass here the functions for example: deleteTodo, completeTodo, editComplete, instead of the setTodos and todos props